src/main/resources/static/js/subscription.js

```diff
@@ -389,7 +389,7 @@ jQuery(document).ready(function () {
                     "data": null,
                     "render": function (data, type, row, meta) {
                         if (data == undefined || row == undefined) {
-                            window.logMessages("Error: Subscription data is not defined");
+                            logMessage("Error: Subscription data is not defined");
```

Comment:
Keep the log messages consistent and remove the Error in the beginning of the sentence.

Reply:
Done.